lib/jsonapi_compliable/adapters/active_record/many_to_many_sideload.rb

```diff
@@ -0,0 +1,34 @@
+module JsonapiCompliable
+  module Adapters
+    module ActiveRecord
+      class ManyToManySideload < Sideload::ManyToMany
```

Comment:
I'd like a generic `through` association but I'm not sure how feasible it is. We may need this *and* `through. Keeping it for now.

Reply:
Any reason not to name this `has_and_belongs_to_many` to mirror AR?

Reply:
Less verbose, but most importantly `HABTM` isn't supported (we require `through:` atm). I think ManyToMany is better for everything but AR, and there it's probbaly more confusing than anything else right now.